src/macrofier/types.mjs

```diff
@@ -251,6 +282,7 @@ const insertTupleMacros = (content, schema, module, title, options) => {
   const itemsTemplate = getTemplate(path.join(options.templateDir, 'items'))
   const propIndent = (content.split('\n').find(line => line.includes("${properties}")) || '').match(/^\s+/) || [''][0]
   const itemsIndent = (content.split('\n').find(line => line.includes("${items}")) || '').match(/^\s+/) || [''][0]
+  const delimiter = getTemplate(path.join(options.templateDir, 'delimiter'))
```

Comment:
`delimiter` is a pretty generic name. Maybe `tuple-delimiter` would be safer?

Reply:
done